lib/clients/graphql/storefront_client.ts

```diff
@@ -27,13 +25,15 @@ export class StorefrontClient extends GraphqlClient {
     this.storefrontAccessToken = params.storefrontAccessToken;
   }
 
-  protected getAccessTokenHeader(): AccessTokenHeader {
+  protected getApiHeaders(): HeaderParams {
     return {
-      header: ShopifyHeader.StorefrontAccessToken,
-      value: (this.storefrontClass().CONFIG.isPrivateApp
+      [ShopifyHeader.StorefrontAccessToken]: this.storefrontClass().CONFIG
+        .isPrivateApp
         ? this.storefrontClass().CONFIG.privateAppStorefrontAccessToken ||
           this.storefrontAccessToken
-        : this.storefrontAccessToken) as string,
+        : this.storefrontAccessToken,
+      [ShopifyHeader.StorefrontSDKVariant]: '@shopify/shopify-api',
```

Comment:
Should this also be a constant that lives next to `SHOPIFY_API_LIBRARY_VERSION` ?

Reply:
That's a good point. I wouldn't put it along with the version, but we should use the same value we use in our User-Agent headers. I'll centralize this.